enterprise/dev/ci/internal/ci/pipeline.go

```diff
@@ -99,7 +99,10 @@ func GeneratePipeline(c Config) (*bk.Pipeline, error) {
 				ops.Append(prPreview())
 			}
 		}
-		ops.Merge(CoreTestOperations(c.Diff, CoreTestOperationsOptions{MinimumUpgradeableVersion: minimumUpgradeableVersion}))
+		ops.Merge(CoreTestOperations(c.Diff, CoreTestOperationsOptions{
+			MinimumUpgradeableVersion: minimumUpgradeableVersion,
+			LintOnlyChangedFiles:      c.RunType.Is(runtype.PullRequest),
```

Comment:
```suggestion
			ClientLintOnlyChangedFiles:      c.RunType.Is(runtype.PullRequest),
```

I think that would be clearer this way, to avoid creating the assumption that we're applying this logic on everything!